code/src/GoogleProvider/DrawingTools/AbstractDrawShape.ts

```diff
@@ -30,13 +33,18 @@ namespace GoogleProvider.DrawingTools {
                         // EventName
                         this.completedToolEventName,
                         // The extra parameters, uniqueId and isNewElement set to false indicating that the element is not new
-                        { uniqueId: _shape.uniqueId, isNewElement: false }
+                        {
+                            uniqueId: _shape.uniqueId,
+                            isNewElement: false,
+                            locations
+                        }
                     );
                 }
             );
         }
 
         /** Create the new shape element based on the configurations (already contains the locations, the bounds or the center and radius depending on the type of the new shape) */
+        // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
This is not needed if the setOnChangeEvent method is in the right place

Reply:
Yup, fixed!

Reply:
Yup, fixed!